book/listings/gobject_values/1/main.rs

```diff
@@ -1,18 +1,36 @@
 use gtk::prelude::*;
 
 fn main() {
-    let number_value = 10.to_value();
-    let string_value = "Hello!".to_value();
+    // ANCHOR: i32
+    // Store `Option<String>` as `Value`
+    let integer_value = 10.to_value();
 
-    // Retrieve `i32` from `number_value`
-    let number = number_value
+    // Retrieve `String` from `Value`
+    let integer = integer_value
         .get::<i32>()
         .expect("The value needs to be of type `i32`.");
-    assert_eq!(number, 10);
+    assert_eq!(integer, 10);
+    // ANCHOR_END: i32
+
+    // ANCHOR: string
+    // Store string as `Value`
+    let string_value = "Hello!".to_value();
 
-    // Retrieve `String` from `string_value`
+    // Retrieve `String` from `Value`
     let string = string_value
         .get::<String>()
         .expect("The value needs to be of type `String`.");
     assert_eq!(string, "Hello!");
+    // ANCHOR_END: string
+
+    // ANCHOR: string_none
+    // Store `Option<String>` as `Value`
+    let string_none_value = None::<String>.to_value();
+
+    // Retrieve `String` from `Value`
+    let string_none = string_none_value
+        .get::<Option<String>>()
```

Comment:
Maybe like this it's not clear that you could also do `string_value.get::<Option<String>>()`

Reply:
Yeah maybe, suggestions are welcome.

Reply:
You could add that line to the example

Reply:
Done